packages/cli/src/cmds/cmd_build/mod.rs

```diff
@@ -131,7 +175,7 @@ fn run_build(args: CmdBuild) -> cu::Result<()> {
     // TODO: Generate cxxbridge headers and sources
     // generate_cxx_bridge_src(rust_crate.src_dir, module_target_path)
 
-    let rust_crate = RustCrate::new(PathBuf::from("packages/modules"));
+    let rust_crate = RustCrate::new(PathBuf::from(config.cargo.manifest.unwrap()));
```

Comment:
IIRC the default manifest should be ./Cargo.toml relative to Megaton.toml

We can review and finalize the config spec in next meeting